ts/features/bonus/cgn/screens/merchants/CgnMerchantDetailScreen.tsx

```diff
@@ -68,154 +59,128 @@ const CgnMerchantDetailScreen = () => {
       <DiscountListItem item={discount} key={index} />
     ));
 
+  const handlePressMerchantWebsite = (websiteUrl?: string) => {
+    if (websiteUrl !== undefined) {
+      openWebUrl(websiteUrl, () =>
+        IOToast.error(I18n.t("bonus.cgn.generic.linkError"))
+      );
+    }
+  };
   // -------    render
 
+  useHeaderSecondLevel({
+    title: "",
+    supportRequest: true
```

Comment:
```suggestion
    title: "",
    canGoBack: true,
    supportRequest: true
```

Just to be more explicit